quirks/password-rules.json

```diff
@@ -380,6 +380,9 @@
     "fuelrewards.com": {
         "password-rules": "minlength: 8; maxlength: 16; allowed: upper,lower,digit,[!#$%@];"
     },
+    "gamefaqs.gamespot.com": {
+        "password-rules": "minlength: 8; required: lower; required: upper; required: digit; allowed: ascii-printable;"
+    },
```

Comment:
I think this rule qualifies as "particularly standard and obvious for password managers." Contributions to password rules should be limited to services [which don't accept a password generated by your password manager with its default settings or reasonable configuration](https://github.com/apple/password-manager-resources/blob/main/CONTRIBUTING.md#contributing-password-rules).

Reply:
Agreed. We should remove this.